include/boost/url/impl/pct_encoded_view.ipp

```diff
@@ -37,6 +37,18 @@ operator*() const noexcept ->
              unsigned char>(d1))));
 }
 
+// unchecked constructor
+pct_encoded_view::
+pct_encoded_view(
```

Comment:
This one can be out of line, with DLL linkage

Reply:
Isn't it already out of line with DLL linkage?

Reply:
I'm just agreeing that it is good the way it is

Reply:
Oh OK.